lighthouse/classes/events/beckman/__init__.py

```diff
@@ -1,7 +1,7 @@
 from .destination_created import DestinationCreated
 from .destination_failed import DestinationFailed
 from .source_all_negatives import SourceAllNegatives
-from .source_completed import SourceCompleted
+from .source_completed import SourceCompleted  # type: ignore
```

Comment:
We shouldn't need this here?

Reply:
Did this get fixed?

Reply:
To fix this I need to make changes in beckman. I was waiting to see if biosero is done.